spec/lib/rbac/filterer_spec.rb

```diff
@@ -2464,19 +2464,22 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
     let(:project1_user)         { FactoryBot.create(:user, :miq_groups => [project1_group]) }
     let(:project1_volume)       { FactoryBot.create(:cloud_volume, :ext_management_system => ems_openstack, :cloud_tenant => project1_cloud_tenant) }
     let(:project1_flavor)       { FactoryBot.create(:flavor, :ext_management_system => ems_openstack) }
+    let(:project1_orchestration_stack) { FactoryBot.create(:flavor, :ext_management_system => ems_openstack) }
```

Comment:
Is a `flavor` a kind of `orchestration_stack`?

Expected something closer to:

```suggestion
    let(:project1_orchestration_stack) { FactoryBot.create(:orchestration_stack, :ext_management_system => ems_openstack) }
```

Reply:
Thanks for help :) Didn't notice this, I'm just too tired by the end of week.